PUBLISHING.md

```diff
@@ -0,0 +1,53 @@
+# Publishing
+
+Publishing this package is as easy as executing these two commands:
+```
+vsce package
+vsce publish
+```
+
+But in order to be able to publish this extension, some preparations have to be made.
+
+## Get a Personal Access Token
+
+1. Navigate to `https://dev.azure.com/process-engine`.
```

Comment:
Bitte nicht in Backticks, ich möchte das nicht rauskopieren müssen, sondern lieber anklicken.

```suggestion
1. Navigate to https://dev.azure.com/process-engine
```